internal/provisioner/plain/controllers/bundle_controller.go

```diff
@@ -229,6 +223,32 @@ func updateStatusUnpackFailing(u *updater.Updater, err error) error {
 	return err
 }
 
+func (r *BundleReconciler) validate(bundle fs.FS) error {
+	var validationErrors []error
+	objects, err := getObjects(bundle)
+	if err != nil {
+		validationErrors = append(validationErrors, fmt.Errorf("get objects from bundle manifests: %v", err))
+	}
+	if len(objects) == 0 {
```

Comment:
`else if` here?

I assume that if we encounter an error getting the objects, the length of `objects` will always be 0.